src/core.c/Mu.pm6

```diff
@@ -11,6 +11,10 @@ my class Mu { # declared in BOOTSTRAP
 
     method sink(--> Nil) { }
 
+    # This method is only here so that it can be introspected.  The codegen
+    # will actually generate an nqp::what in all cases.
+    method WHAT() { self.WHAT }
```

Comment:
What if codegen changes later? Perhaps it worth just having `nqp::what` here despite it won't be invoked anyway? Just for the safety of it.

Reply:
If that codegen changes, then this should change as well, wouldn't it?

Reply:
Sure. But my point is that it could be overlooked. I'd prefer something that won't silently go into deep recursion.

Reply:
So you're implying that the method would actually be called at some point ?

Reply:
Exactly. And it's not only due to potentially changed codegen. It could be invoked indirectly, or via '$obj."$what-method-name"()`, etc.

Reply:
The code-gen change risk is low, but I'm a bit uncomfortable with this; I'd prefer that the name suggestion code carried the small list (`WHAT`, `HOW`, and `WHO`) of things that are true MOP primitives rather than adding these methods.